notification-portlet-webapp/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -120,7 +120,20 @@
                 <name>usePortalJsLibs</name>
                 <value>false</value>
             </preference>
-
+            
+            <!-- Indicates whether the portlet will mark a notification as 'COMPLETE' when
```

Comment:
Should also mention that it only works for notifications from the JPA service (e.g. stored in the database) at this point.